vowpalwabbit/cb_explore.cc

```diff
@@ -286,9 +285,9 @@ void output_example(vw& all, cb_explore& data, example& ec, CB::label& ld)
   print_update_cb_explore(all, CB::cb_label.test_label(&ld), ec, sso);
 }
 
-void finish_example(vw& all, cb_explore& c, example& ec)
+void finish_example(vw& all, cb_explore&, example& ec)
```

Comment:
will the copied examples in `adf_data.ecs` be cleared for re-use here too?

Reply:
I'm actually blindly trusting in that calling copy again on the next run would bring those examples to a good state from the get go. I'll verify if this is the case.

Reply:
not needed, the copy fn calls into default_label which calls into clear. Let me know if this is what you meant.